doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+   |                              | Layer Output Bits     |         3         |       5          |         5         |       5          |
+   +------------------------------+-----------------------+-------------------+------------------+-------------------+------------------+
+   |                              | MACs per Output Value |   76 (3*5*5+1)    | 801 (64*16+1)    | 401 (16*5*5+1)    | 513 (32*16+1)    |
+   |                              +-----------------------+-------------------+------------------+-------------------+------------------+
+   | **Accumulator Restrictions** | Required Extra Bits   |         7         |      10          |         9         |      10          |
+   |                              +-----------------------+-------------------+------------------+-------------------+------------------+
+   |                              | Not Enough Bits       |         0         |   10 – 9 = 1     |      9 – 9 = 0    |   10 – 9 = 1     |
+   +------------------------------+-----------------------+-------------------+------------------+-------------------+------------------+
+   |                              | Layer Input Bits      |         0         |    3 + 1 = 4     |         5         |    5 + 1 = 6     |
+   |                              | (updated)             |                   |                  |                   |                  |
+   | **Integer Bit Requirements** +-----------------------+-------------------+------------------+-------------------+------------------+
+   | *(fx16 operands)*            | Layer Weight Bits     |        -1         |      -1          |        -1         |      -1          |
+   |                              | (updated)             |                   |                  |                   |                  |
+   |                              +-----------------------+-------------------+------------------+-------------------+------------------+
+   |                              | Layer Output Bits     | 4 (next layer in) |       5          | 6 (next layer in) |       5          |
+   |                              | (updated)             |                   |                  |                   |                  |
+   +------------------------------+-----------------------+-------------------+------------------+-------------------+------------------+
+
+   
+.. note::
+   Defining Q format in this way, you can guarantee that accumulator is not saturated while a single output is being calculated. But the restriction may be loosened if you are sure about your data. For example, look at the final fully connected layer above: 9 bits are enough if we do not consider bias addition. Analyze how likely is it that for 1 extra addition result will overflow the defined range. Moreover, saturation of results might have a minor effect on the network accuracy. 
```

Comment:
"... For example, look at the final fully connected layer above: 9 bits are enough if we do not consider bias addition. Analyze how likely is it that for 1 extra addition result will overflow the defined range. " ->  "... For example, look at the final fully connected layer above: 9 bits (for 512 MACs) are enough if we do not consider bias addition. "